packages/syft/src/syft/core/node/new/user_code.py

```diff
@@ -252,7 +397,8 @@ class UserCode(SyftObject):
     user_unique_func_name: str
     code_hash: str
     signature: inspect.Signature
-    status: UserCodeStatus = UserCodeStatus.SUBMITTED
+    status: UserCodeStatusContext
+    enclave_metadata: Optional[EnclaveMetadata] = None
```

Comment:
This is enclave specific, i think we should consider a different or more generic way to store this.

Reply:
The is mainly set , when we work with an Enclave Client, 
and is set to None when we work with Domain Client

I mainly  to wanted to reduce different code paths and optimise UI flow,
Let me if you have any other idea 🙂 , we could modify it